crates/esthri/src/ops/download.rs

```diff
@@ -67,20 +69,31 @@ where
     download_file(s3, bucket.as_ref(), key.as_ref(), file.as_ref(), true).await
 }
 
-#[logfn(err = "ERROR")]
```

Comment:
I got rid of this because it was causing type errors. I didn't look into it too much but I guess the macro has difficulty parsing the types.

Reply:
I've tricked it into working by having a wrapper function and having the annotation on the wrapper: https://github.com/swift-nav/esthri/pull/276/commits/2247f2763372d571470b9c956d35132cc5d68903 . I don't really understand the original error but this seems to work at least. :man_shrugging: